server/dataGeneration/index.js

```diff
@@ -1,6 +1,5 @@
 require('./generateDumpHead');
-require('./generateCopySpaces');
 require('./generateCopyReviews');
-require('./generateCopyCalculations');
+require('./generateCopyAuthors');
```

Comment:
I noticed this is all the file is comprised of. Do these run sequentially from a script? Just curious to understand what the purpose of this file is.

Reply:
The `npm run data-generate` script runs this file so that all of these files are called in order